capstone/src/main/java/com/google/sps/data/FeatureData.java

```diff
@@ -0,0 +1,129 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+/* This class stores data on various features in the model */
+public class FeatureData {
+  private static ArrayList<Precinct> precinctData;
+
+  private final int NUMBER_OF_PRECINCTS = 10;
+
+  /* The precinct data was taken from public datasets such as
+  https://www.incomebyzipcode.com/california/94111 and
+  https://www.sanfranciscopolice.org/stay-safe/crime-data/crime-dashboard */
+  static {
+    precinctData = new ArrayList<Precinct>();
+
+    Precinct southern = new Precinct();
+    southern.setName("Southern");
+    southern.setPopulation(40384);
+    southern.setAverageHouseholdIncome(153727.5f);
+    southern.setCrimeRate(calculateRate(72, 40384));
+    southern.setPoliceStationRating(3.0f);
+
+    Precinct mission = new Precinct();
```

Comment:
Consider the builder pattern here in the future: https://stackoverflow.com/questions/328496/when-would-you-use-the-builder-pattern

Reply:
Thank you for this tip, this is an interesting pattern, I will add the fix :)